tests/postgres/mod.rs

```diff
@@ -285,39 +287,49 @@ async fn test_postgres_jsonb_type(port: usize) {
 
     let schema = Arc::new(Schema::new(vec![Field::new("data", DataType::Utf8, true)]));
 
-    // Parse and re-serialize the JSON to ensure consistent ordering
     let expected_values = vec![
-        serde_json::from_str::<Value>(r#"{"name":"John","age":30}"#)
-            .unwrap()
-            .to_string(),
-        serde_json::from_str::<Value>(r#"{"name":"Jane","age":25}"#)
-            .unwrap()
-            .to_string(),
-        serde_json::from_str::<Value>("[1,2,3]")
-            .unwrap()
-            .to_string(),
-        serde_json::from_str::<Value>("null").unwrap().to_string(),
-        serde_json::from_str::<Value>(r#"{"nested":{"key":"value"}}"#)
-            .unwrap()
-            .to_string(),
+        r#"{"name": "John", "age": 30}"#,
+        r#"{"name": "Jane", "age": 25}"#,
+        "[1, 2, 3]",
+        "null", 
+        r#"{"nested": {"key": "value"}}"#,
```

Comment:
This test was failing due to order of order of the fields in the string. I refactored it to compare json values directly, not strings.

Not sure why it happened. Maybe some library version got updated.